packages/core/index.test-d.ts

```diff
@@ -3,10 +3,19 @@ import middy from '.'
 import {
   APIGatewayProxyEvent,
   APIGatewayProxyResult,
-  Context,
-  Handler as LambdaHandler
+  Callback,
+  Context
 } from 'aws-lambda'
 
+// extended Handler type from aws-lambda
+// to include synced TResult
+type LambdaHandler<TEvent = any, TResult = any> = (
```

Comment:
I think an ideal solution is to somehow extend the return type of the original `Handler` type from aws. Looking to see if that's even possible.